packages/chat-component/src/main.ts

```diff
@@ -133,7 +164,7 @@ export class ChatComponent extends LitElement {
     if (isUserMessage) {
       updateChatWithMessageOrChunk(message, false);
     } else {
-      if (this.isStreaming) {
+      if (this.useStream) {
```

Comment:
conventionally booleans are named isName or hasName...do we really want to change that?